runtime/src/configs/xcm_config.rs

```diff
@@ -1,4 +1,29 @@
-use super::{
+// This is free and unencumbered software released into the public domain.
```

Comment:
As this is in a config module now, can we not jsut call it xcm?

Reply:
Fixed in last commit